<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrepearTransferWidget</class>
 <widget class="QDialog" name="PrepearTransferWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>424</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Перенос частицы</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_9">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="4" column="0" colspan="3">
      <widget class="QTreeWidget" name="fieldsTree">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="addBtn">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="closeBtn">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="7" column="1">
        <widget class="QComboBox" name="ccenterComboBox">
         <property name="maxCount">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="timeBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>7</number>
         </property>
         <property name="maxCount">
          <number>999</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>36</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>72</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>96</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>120</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>144</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Уровень</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Центр</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="koefBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="maxCount">
          <number>999</number>
         </property>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.1</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="fieldBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <property name="maxCount">
          <number>99999</number>
         </property>
         <item>
          <property name="text">
           <string>925</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>850</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>700</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Время переноса (ч)</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="intervalBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>5</number>
         </property>
         <property name="maxCount">
          <number>999</number>
         </property>
         <item>
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Интервал печати (ч)</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Коэффициент пересчета</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Текущая дата:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QDateEdit" name="dateEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Время:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTimeEdit" name="timeEdit">
             <property name="calendarPopup">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QToolButton" name="labelBtn">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Траектория:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="forwardRadio">
             <property name="text">
              <string>Прямая</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QRadioButton" name="backRadio">
             <property name="text">
              <string>Обратная</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Поля, используемые для расчетов:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="fieldBtn">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
